accessibility.html

```diff
@@ -0,0 +1,281 @@
+              <h2>Accessibility guidelines</h2>
+              
+              <h3>Keyboard accessibility</h3>
+              <p>
+                A user should be able to navigate to all content using a keyboard. 
+                They should be able to use the TAB key to get to all inputs, links and buttons. 
+                They should be able to submit forms using the ENTER key. 
+                They should be able to open and close panels using the ENTER and ESC keys.
+              </p>
+              <p>
+                Feedback should be provided to the user when they TAB to elements, with the use of a focus states.              
+              <p>
+                
+              <h3>All content must be easy to understand</h3>
+              <p>
+                All writing should use simple language. 
+                Technical terms should be simplified where possible. 
+                Each part of the content should have an appropriate heading.
+              </p>
+              
+              <h3>Semantic HTML</h3>
+              <p>
+                HTML5 is wonderful. With its introduction we got many elements that gave actual context to what they do (<em>&lt;header&gt;, &lt;article&gt;, &lt;footer&gt;</em>), 
+                reducing the need to use lots of <em>&lt;div&gt;</em> elements that give no semantic understanding.
+              </p>
+              
+              <h3>ARIA Roles</h3>
+              <p>
+                All HTML Elements must have <a href="http://a11yproject.com/posts/getting-started-aria">ARIA Roles</a> where needed.
+                ARIA Roles tell assistive devices like screenreaders (JAWS, VoiceOver) the role of the element (is it a header, is it a footer, is it the main content?) 
+                and its state (hidden, visible, open, closed, etc.).
+              </p>
+              <p>
+                With HTML5, many ARIA roles are implicit to the semantic element. So in theory, you do not need to add <em>role='header'</em> to a <em>&lt;header&gt;</em> element. 
+                But, as not all browsers and technologies support these HTML elements, it is a good idea right now to add ARIA roles to our elements whether the role is already implicitly implied or not.
+              </p>
+              
+              <h3>Design</h3>
+              <p>Design is important. We must make our websites as visually accessible as possible.</p>
+              <h4>Contrast Ratio</h4>
+              <p>
+                We must be able to make the out the text of an element from its background. 
+                A dark grey element with slightly lighter grey text is difficult to decipher.
+              </p>
+              <h4>Hover, focus, active states</h4>
+              <p>
+                A user must know when they are interacting with something. All links, buttons and other focusable elements must stand out through the use of colours and underlines. They should also have appropriate hover, 
```

Comment:
"use of colours and underlines" It doesn't need to be underlines, but it must be something other than just color. Imagine you have a block that is 100px square that is focusable, are you going to underline it?